### ISSUE TITLE
Issue while installing mongodb-beta using pecl

### ISSUE THREAD

**Issue by susonwaiba** [mongodb/mongo-php-driver#1147]
### Bug Report

While installing mongodb-beta using `sudo pecl install mongodb-beta`
It seems, downloading 1.7.5

```
...
downloading mongodb-1.7.5.tgz ...
Starting to download mongodb-1.7.5.tgz (1,223,892 bytes)
```

### Environment

- Ubuntu 20.04
- PHP-7.4

### Expected and Actual Behavior

- Expected to download 1.8.0 but it seems downloading 1.7.5

**Comment by jmikola** [comment]
Unfortunately, PEAR/PECL prioritize the most recent release over any stability preferences. Since 1.7.5 was released _after_ 1.8.0beta2, it is selecting that stable release over your preference for a beta release. The only workaround to my knowledge is to provide the explicit version that you wish to install:

```
pecl install mongodb-1.8.0beta2
```

I've made a note of this in [PHPC-1660](https://jira.mongodb.org/browse/PHPC-1660), so we can ensure this is followed going forward. In the meantime, I will go back and revise the `pecl install` examples in the existing release notes for the 1.8.0 beta releases.

Thanks for the report, as we'll be able to address this going forward.
